CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+<!-- SPDX-License-Identifier: CC0-1.0 -->
+# Contributing to Govdirectory
+
+🙇‍♀️ Thank you for contributing!
+
+We love issues and pull requests from everyone.
+If you're not comfortable with those features of GitHub, you can also just [start a discussion](https://github.com/govdirectory/website/discussions).
+
+## Problems, suggestions and questions in issues
+
+Please help development by reporting problems, suggesting changes and asking questions.
+To do this, you can [create a GitHub issue](https://help.github.com/articles/creating-an-issue/) for this project in the [GitHub Issues for the website](https://github.com/govdirectory/website/issues).
```

Comment:
We should probably point people directly to https://github.com/govdirectory/website/issues/new and there provide one or more issue templates that should be self explanatory.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates#issue-templates

Reply:
Yes, I'll change the link, but I don't think having templates should be a blocker.

Reply:
```suggestion
To do this, you can [create a GitHub issue](https://help.github.com/articles/creating-an-issue/) for this project in the [GitHub Issues for the website](https://github.com/govdirectory/website/issues/new).
```